services/ui-src/src/forms/addEditMcparReport/addEditMcparReport.ts

```diff
@@ -0,0 +1,105 @@
+import { ProgramList } from "../../constants";
+import { DropdownOptions } from "types";
+
+// create dropdown options
+const dropdownOptions: DropdownOptions[] = Object.keys(ProgramList).map(
+  (value) => {
+    return {
+      label: ProgramList[value as keyof typeof ProgramList],
```

Comment:
This feels reminiscent of when QMR tied the labels to the ids. Not sure if it'll be the same issue here but something to note.